### PR TITLE
First attempt at documenting use of PCI_PNP_INFO, PCI_DEV, PCI_SUBDEV, PCI_VID, PCI_DEVID

### PR DESCRIPTION
**Pull request by lakhanshiva**
Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy lakhanshiva@gmail.com
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: #14
Url: https://reviews.freebsd.org/

### CODE REVIEW

**Comment by ctuffli** [comment]
Should the file names end with a `.9`? For example, should ` share/man/man9/PCI_DEVID` be ` share/man/man9/PCI_DEVID.9`?

**Comment by lakhanshiva** [comment]
Every file in here has a .9 in its file name at the end. https://github.com/lakhanshiva/freebsd/tree/lakhan-pnpinfo-new/share/man/man9/. If we look at this for example, https://www.freebsd.org/cgi/man.cgi?query=MODULE_PNP_INFO&manpath=FreeBSD+12-current, i understand that the reason for this is because the driver is represented as MODULE_PNP_INFO(9) in the first line. Correct me if i am wrong.

**Comment by ctuffli** [comment]
Weird. When I reviewed this yesterday, it appeared the `.9` was missing from several of the new files.

**Comment by lakhanshiva** [comment]
You are right. I fixed it only yesterday.

File: share/man/man9/PCI_DEV.9 [hunk 1]
```diff
@@ -0,0 +1,84 @@
+.\" Copyright (c) 2018 Lakhan Shiva Kamireddy <lakhan@FreeBSD.org>
+.\" Sponsored by: Google, Inc. (GSoC 2018)
+.\" Mentors: Warner Losh, Chuck Tuffli
+.\" All rights reserved.
```
Comment (bsdimp) [inline share/man/man9/PCI_DEV.9]: These three lines aren't needed. We don't document this stuff in the man pages. All Rights Reserved isn't useful anymore.

File: share/man/man9/PCI_DEV.9 [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+.Sh HISTORY
+The macro
+.Nm
+appeared in
+.Fx 12.0 .
+.Sh AUTHORS
+The PNP framework and
+.Xr devmatch 8
+utility were written by
+.An Warner Losh Aq Mt imp@FreeBSD.org .
```
Comment (bsdimp) [inline share/man/man9/PCI_DEV.9]: Give yourself credit here.

**Review by bsdimp — APPROVED** [review]

**Pull request merged**